core/src/main/scala_2.10/macrocompat/macrocompat.scala

```diff
@@ -200,6 +200,8 @@ trait MacroCompat {
     def dealias: Type = tpe.normalize
 
     def finalResultType: Type = (tpe: global.Type).finalResultType
+
+    def paramLists: List[List[global.Symbol]] = tpe.paramss
```

Comment:
Yeah.. seems `paramss` is only on `scala.tools.nsc.Global.Type`.. need to cast it back.

Updated the pull request, what do you think?